lib/cli/run.js

```diff
@@ -190,7 +192,7 @@ exports.builder = yargs =>
             const pair = opt.split('=');
 
             if (pair.length > 2 || !pair.length) {
-              throw new Error(`invalid reporter option '${opt}'`);
+              throw createInvalidReporterError('invalid reporter option', opt);
```

Comment:
Said this last time. This is not an `InvalidReporterError`, but a `TypeError` with code `ERR_INVALID_ARGUMENT_VALUE`. This is CLI parsing code...

Reply:
Ok will add `createInvalidArgumentTypeError`. Also have been using mocha prefix even for type errors so `ERR_MOCHA_INVALID_ARG_VALUE `.

Reply:
Note that this one would use `createInvalidArgumentValueError`.